docs/Developers/Profile_Bears.rst

```diff
@@ -0,0 +1,74 @@
+Profile Bears
+=============
+
+This document provide the overview of coala's Profiling Interface. The
+Profiling Interface will provide an ability to Profile the Bear's code to
```

Comment:
I think plural sounds better:

The Profiling Interfaces provides abilities to ~~P~~profile Bear~~s~~ code